### DISCUSSION TITLE
Building wix tools

### DISCUSSION THREAD

**Discussion by DevX-Cipher** [wixtoolset/issues#8202] [Questions]
hi im trying to build wix tools form the command line too not sure why it fails but here is the report 

C:\Users\Modz2014\Downloads\wix3-develop\wix3-develop\tools\OneTimeWixBuildInitialization.proj(36,5): error : Could not
 find sn.exe. Expected to find it at: \bin\sn.exe
Done Building Project "C:\Users\Modz2014\Downloads\wix3-develop\wix3-develop\tools\OneTimeWixBuildInitialization.proj"
(default targets) -- FAILED.

Build FAILED.

"C:\Users\Modz2014\Downloads\wix3-develop\wix3-develop\tools\OneTimeWixBuildInitialization.proj" (default target) (1) -
>
(RegisterStrongNameSkip target) ->
  C:\Users\Modz2014\Downloads\wix3-develop\wix3-develop\tools\OneTimeWixBuildInitialization.proj(36,5): error : Could n
ot find sn.exe. Expected to find it at: \bin\sn.exe

    0 Warning(s)
    1 Error(s)

**Comment by DevX-Cipher** [comment]
@robmen dont seem to know what the problem is

**Reply by robmen** [reply]
@modz2014 I'm focused on customer needs right now. Best of luck to you.

**Reply by DevX-Cipher** [reply]
so you whos going to help fix the issue of not being able to build the project how can i add feature if i cant build the project correctly

**Comment by chrpai** [comment]
FWIW, I doubt WiX team would be super interested in taking PRs for the v3 baseline. They are focused on v6 now.  Fortunately the v4-5-6 code is a lot easier to build.

**Reply by DevX-Cipher** [reply]
ok i havent looked elsewhere im interested in dark.exe
